gulpfile.mjs

```diff
@@ -491,7 +491,10 @@ function checkChromePreferencesFile(chromePrefsPath, webPrefs) {
 
 function createMainBundle(defines) {
   const mainFileConfig = createWebpackConfig(defines, {
-    filename: defines.MINIFIED ? "pdf.min.mjs" : "pdf.mjs",
+    filename:
+      defines.MINIFIED && !defines.MOZCENTRAL && !defines.GECKOVIEW
+        ? "pdf.min.mjs"
+        : "pdf.mjs",
```

Comment:
Can't we update to use the new filename in the HTML instead?
 - https://github.com/mozilla/pdf.js/blob/b070a502c33aa08788e31e8f2afb675df1405374/web/viewer.html#L48
 - https://github.com/mozilla/pdf.js/blob/b070a502c33aa08788e31e8f2afb675df1405374/web/viewer-geckoview.html#L95

Reply:
Why not ? but why not keeping the same names ?